client/generated_backing_image.go

```diff
@@ -11,9 +11,13 @@ type BackingImage struct {
 
 	DiskStateMap map[string]string `json:"diskStateMap,omitempty" yaml:"disk_state_map,omitempty"`
 
+	DownloadProgressMap map[string]string `json:"downloadProgressMap,omitempty" yaml:"download_progress_map,omitempty"`
```

Comment:
why is this in your commit/pr?

Reply:
I don't know, it got automatically updated with the generator. I assumed it's something we've missed to update, and fine to keep?

Reply:
I think Chin-Ya just directly ran the update script and put all `client` modifications in one commit

Reply:
Exactly what I did. Should this be removed?

Reply:
No. You can rename the commit title to `Update package "client" to the latest`. 
BTW, there is no need to name the commit title as `Implement auto rebalance - xxxxxx`. The commit separation I mentioned before aims to split a big commit into smaller commits based on the functionality/package you modified. It doesn't mean that the title naming schema is fixed.

Reply:
Thanks, addressed.